src/CMakeLists.txt

```diff
@@ -56,6 +56,8 @@ if(WIN32)
 endif()
 
 if(MSVC)
+    enable_language(CXX)
```

Comment:
I don't understand why that is needed.
After all, AVRDUDE is a C-only project, so why enable C++?

Reply:
This is the side effect for the MSVC because of the changes in the other CMakeList.txt file. If without this change, MSVC build will fail.

Reply:
> After all, AVRDUDE is a C-only project, so why enable C++?

I used C++ for some of the MSVC shims, the libwinusb library is mostly C++.

PR is fine by me.